app/views/state_file/questions/eligible/_md_supported.html.erb

```diff
@@ -1,7 +1,7 @@
 <h3 class="h3"><%= t('.heading1') %></h3>
 <ul class="list--bulleted">
   <li><%= t('.sub1') %></li>
-  <li><%= t('.sub2') %></li>
+  <li><%= t('.sub2', count: current_intake.eligibility_filing_status_mfj_yes? ? 1 : 2) %></li>
```

Comment:
`yes` would be count of 2 not 1

Reply:
Ah yes I have it backwards twice. Good catch.